packages/devextreme/scss/widgets/fluent/tabs/variables/_colors.scss

```diff
@@ -10,31 +10,61 @@ $fluent-tabs-nav-button-background: color.change($button-default-hover-bg, $alph
 * $name 10. Tab text color
 * $type color
 */
-$tabs-tab-color: color.change($base-text-color, $alpha: 0.54) !default;
+$tabs-tab-color: $base-text-color !default;
 
 /**
 * $name 20. Selected tab text color
 * $type color
 */
-$tabs-tab-selected-color: $base-accent !default;
+$tabs-tab-selected-color: $base-text-color !default;
 
 /**
 * $name 30. Disabled tab color
 * $type color
 */
-$tabs-tab-disabled-color: color.change($tabs-tab-color, $alpha: 0.32) !default;
+$tabs-tab-disabled-color: $base-color-disabled !default;
 
 /**
 * $name 40. Selected tab border color
 * $type color
 */
-$tabs-selected-tab-border-color: $base-accent !default;
+$tabs-tab-border-color-selected-basic: $base-accent !default;
+
+/**
+* $name 41. Selected active tab border color
+* $type color
+*/
+$tabs-tab-border-color-selected-active: $base-accent-active !default;
+
+/**
+* $name 42. Selected hover tab border color
+* $type color
+*/
+$tabs-tab-border-color-selected-hover: $base-accent-hover !default;
+
+/**
+* $name 43. Selected focused tab border color
+* $type color
+*/
+$tabs-tab-border-color-selected-focused: $base-accent !default;
+
+/**
+* $name 44. Active tab border color
+* $type color
+*/
+$tabs-tab-border-color-active: $base-border-color-focused !default;
+
+/**
+* $name 42. Hover tab border color
+* $type color
+*/
+$tabs-tab-border-color-hover: $base-border-color-hover !default;
```

Comment:
Are u sure we should publish all these new variables?

Reply:
Why not? It would be better if you could tell me what kind of variables we should publish.

Reply:
I would prefer not to publish it. If necessary, it is better to publish the base variables that are used for the 'hover', `focused`, `active` states and the changes will be applied to the components in which they are used. It is redundant to publish the same variables for each component.

Reply:
[[refactor(fluent): Remove extra variable && Remove ways to publish variables](07cb4534d39dda5d2d3359df2bf5fe85dd3cd4ad)]

Reply:
[[refactor(fluent): Remove extra variable && Remove ways to publish variables](154c4f1fffced1986890ef03376011e70927dd60)]